src/autobind/autobind.ts

```diff
@@ -44,7 +45,7 @@ function isProbablySearchField(el: HTMLElement) {
 }
 
 function getEditableElements(doc: Document = document): HTMLElement[] {
-  const visibleElements: HTMLElement[] = _.filter((doc.querySelectorAll(EDITABLE_ELEMENTS_SELECTOR) as any) as List<HTMLElement>, isDisplayed) as HTMLElement[];
+  const visibleElements: HTMLElement[] = _.filter((doc.querySelectorAll(EDITABLE_ELEMENTS_SELECTOR) as any) as List<HTMLElement>) as HTMLElement[];
```

Comment:
@gohaterahul  In my opinion the visibleElements variable makes not much sense anymore and is a bit confusing because it can contain also invisible elements now. Therefore I would propose to remove it and just write
`return _(doc.querySelectorAll(EDITABLE_ELEMENTS_SELECTOR)).flatMap((el: HTMLElement) => {`
directly.